action.yml

```diff
@@ -289,7 +289,10 @@ runs:
           --output "${{ github.workspace }}/atmos-apply-summary.md" \
           --log-level $([[ "${{ inputs.debug }}" == "true" ]] && echo "DEBUG" || echo "INFO") \
           apply -- terraform apply ${{ steps.vars.outputs.plan_file }} || EXIT_CODE=$?
-
+        
+        TERRAFORM_OUTPUTS=$(terraform output --json | jq -Mr 'to_entries | map(["|", .key, "|", if .value.sensitive then "\\<sensitive\\>" else .value.value end, "|"] | join(" ")) | if length > 0 then ["| name | value |", "|----|----|"] + . else ["No outputs"] end | join("\n")')
```

Comment:
```suggestion
        TERRAFORM_OUTPUTS=$(terraform output --json | jq -Mr 'to_entries | map(["|", .key, "|", if .value.sensitive then "![Sensitive](https://img.shields.io/badge/sensitive-c40000?style=for-the-badge)" else .value.value end, "|"] | join(" ")) | if length > 0 then ["| name | value |", "|----|----|"] + . else ["No outputs"] end | join("\n")')
```

Reply:
my preference would be to encapsulate single-line outputs with `inline code`  and multiline values with a 
```
code block
```

As outputs are often maps, which would render better that way.

Also, I have _concerns_ of the mileage we'll get using JQ for formatting. But if we can get what we need... that's okay.

Reply:
Done